WordPress/UITestsFoundation/Screens/MySiteScreen.swift

```diff
@@ -170,4 +178,18 @@ public class MySiteScreen: ScreenObject {
     public static func isLoaded() -> Bool {
         (try? MySiteScreen().isLoaded) ?? false
     }
+
+    @discardableResult
+    public func verifyDomainsCard() -> Self {
+        let cardText = app.staticTexts["Stake your claim on your corner of the web with a site address that’s easy to find, share and follow."]
```

Comment:
I'm wondering if there's a value in the UI test failing if we decide to change the copy?

IMHO, it's important that such a card appears, it's clickable, and the new DomainsScreen appears. If you agree, maybe we could introduce an identifier that could be used for verifying the card appearing but the success of the test would be independent of the copy. What do you think?

Reply:
Good question! I'd say it depends on how often the copy might change after #20570.

IMO, there's some value in checking that the text is not truncated and that it's displayed on screen (since we check for `isHittable` property, not `exists`).

Using an identifier would make things easier, and I think it makes a lot of sense if we expect the copy to change often.

Reply:
We're not planning many changes after https://github.com/wordpress-mobile/WordPress-iOS/pull/20570 but there's a possibility of it changing once in a while. 

> IMO, there's some value in checking that the text is not truncated and that it's displayed on screen (since we check for isHittable property, not exists).

Good point. I'll leave the decision to you then, I don't have a super-strong opinion on this. 

⚠️  I merged #20570 now, so the copy has changed from `Own your online identity with a custom domain` to a shorter `Find a custom domain`

Reply:
I updated from `trunk` and adjusted the copy in d817a7de1a691e6060304e00f7c9281779ff62ba. Waiting for 🟢 CI 🤞